### ISSUE TITLE
`doc/ex/clip_path.rb` doesn't work with ImageMagick 6.9.10

### ISSUE THREAD

**Issue by Watson1978** [rmagick/rmagick#604]
`doc/ex/clip_path.rb` doesn't work with ImageMagick 6.9.10.
But it works with ImageMagick 6.8 and 6.7.

## With ImageMagick 6.9.10
* Environment
```
$ ruby -r rmagick -v -e 'puts Magick::Long_version'
ruby 2.3.8p459 (2018-10-18 revision 65136) [x86_64-darwin18]
This is RMagick 3.1.0 ($Date: 2009/12/20 02:33:33 $) Copyright (C) 2009 by Timothy P. Hunter
Built with ImageMagick 6.9.10-41 Q16 x86_64 2019-04-23 https://imagemagick.org
Built for ruby 2.3.8
Web page: http://rmagick.rubyforge.org
Email: rmagick@rubyforge.org
```

* Result
```
$ cd doc/ex
$ ruby clip_path.rb
clip_path.rb:52:in `draw': non-conforming drawing primitive definition `image' @ error/draw.c/RenderMVGContent/4342 (Magick::ImageMagickError)
	from clip_path.rb:52:in `<main>'
```

## With ImageMagick 6.8.9
* Environment
```
$ ruby -r rmagick -v -e 'puts Magick::Long_version'
ruby 2.3.8p459 (2018-10-18 revision 65136) [x86_64-darwin18]
This is RMagick 3.1.0 ($Date: 2009/12/20 02:33:33 $) Copyright (C) 2009 by Timothy P. Hunter
Built with ImageMagick 6.8.9-10 Q16 x86_64 2019-04-27 http://www.imagemagick.org
Built for ruby 2.3.8
Web page: http://rmagick.rubyforge.org
Email: rmagick@rubyforge.org
```

* Result
```
$ cd doc/ex
$ ruby clip_path.rb
```
![clip_path](https://user-images.githubusercontent.com/199156/56976801-b7953000-6bae-11e9-8b11-d36ee39e2fa8.gif)

**Comment by Watson1978** [comment]
@dlemstra What has changed with ImageMagick 6.9 to fail this example?

**Comment by Watson1978** [comment]
@dlemstra Can you investigate this issue ?

**Comment by dlemstra** [comment]
Should be fixed with #608
